h/static/styles/partials-v2/_lozenge.scss

```diff
@@ -1,3 +1,4 @@
+.special-lozenge,
```

Comment:
I thought the plan was to just use the existing `lozenge` class and avoid defining a new `special-lozenge` class, since we don't need to customize the appearance at this stage.